### PR TITLE
Refer to Pipewire in buffer setting text

### PR DESCRIPTION
**Pull request by gilgongo**
[Cover use of Pipewire in help text](https://github.com/jamulussoftware/jamulus/issues/2347)

CHANGELOG: Client: added instructions for setting buffers when using Pipewire

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [ ] I tested my code and it does what I want
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#source-code-consistency) 
-  [ ] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [ ] I've filled all the content above

### CODE REVIEW

File: src/clientsettingsdlg.cpp [hunk 1]
```diff
@@ -201,7 +201,9 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                      "In this case the buffer delay setting is disabled and has to be "
                                      "changed using the sound card driver. On Windows, use the "
                                      "ASIO Device Settings button to open the driver settings panel. On Linux, "
-                                     "use the JACK configuration tool to change the buffer size." )
+                                     "use the JACK configuration tool to change the buffer size."
+                                     "Other APIs like Pipewire might need different (command line) options. "
+                                     "Please look in their manual.")
```
Comment (henkdegroot) [inline src/clientsettingsdlg.cpp:206]: Minor formatting issue, space needed before the )

File: src/clientsettingsdlg.cpp [hunk 2]
```diff
@@ -224,7 +226,8 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                        "setting from within %1. "
                                        "On Windows, press the ASIO Device Settings button to open the "
                                        "driver settings panel. On Linux, use the JACK configuration tool to "
-                                       "change the buffer size." )
+                                       "change the buffer size. Other APIs like Pipewire might need different "
+                                       "(command line) options. Please look in their manual.")
```
Comment (henkdegroot) [inline src/clientsettingsdlg.cpp:230]: Minor formatting issue, space needed before the )

**Review by henkdegroot — CHANGES_REQUESTED** [review]

**Comment by pljones** [comment]
Is the CodeQL correct?

**Review by henkdegroot — APPROVED** [review]

**Comment by ann0see** [comment]
@pljones Yes, CodeQl looks strange.

File: src/clientsettingsdlg.cpp [hunk 3]
```diff
@@ -201,7 +201,9 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
                                      "In this case the buffer delay setting is disabled and has to be "
                                      "changed using the sound card driver. On Windows, use the "
                                      "ASIO Device Settings button to open the driver settings panel. On Linux, "
```
Comment (ann0see) [inline src/clientsettingsdlg.cpp:203]: On Linux --> If you use JACK (e.g. on Linux)
since it's probably the same for the JACK on Windows version.

**Review by ann0see — COMMENTED** [review]
Just a minor comment. Not to merger: Please squash merge.

File: src/clientsettingsdlg.cpp (hunk 3 above)
Comment (henkdegroot) [reply src/clientsettingsdlg.cpp:203]: I did not check the code now, but believe to remember this text is not showing on Windows with JACK. At least not currently, but might be good to be prepared for possible changes in the future.

Comment (ann0see) [reply src/clientsettingsdlg.cpp:203]: Could be. Thanks for the information. But, I'd say it's still worth changing.

Comment (pljones) [reply src/clientsettingsdlg.cpp:203]: Yes, saying "On Windows" and then _only_ refering to ASIO is wrong.  So it needs changing.  Also, JACK also is not Linux-only.

These would be better phrased as
> Use the appropriate tool for the interface in use to adjust this buffer size.  For example, if using ASIO, use the "ASIO Device Settings" button to open the driver settings panel or if using JACK, use a tool such as qJackCtl to adjust the buffer size.  Other interfaces, such as Pipewire, would require their appropriate tool being used.  Please refer to the interface manual.

**Comment by pljones** [comment]
@gilgongo will this progress for 3.9.0?

**Comment by ann0see** [comment]
Yes. Please write QjackCtl as on the official site https://qjackctl.sourceforge.io/

File: src/clientsettingsdlg.cpp [hunk 4]
```diff
@@ -179,54 +179,54 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
...
+
+    QString strSndCrdBufDelayTT =
+        tr ( "Some sound card drivers do not allow the buffer delay to be changed from within %1. "
+             "In this case the buffer delay setting is disabled and has to be changed using the sound card driver. "
+             "Use the appropriate tool for the interface in use to adjust this buffer size. "
+             "For example, if using ASIO, use the \"ASIO Device Settings\" button to open the driver settings panel or if using JACK, use a tool "
+             "such as QjackCtl to adjust the buffer size. "
+             "Other interfaces, such as Pipewire, would require their appropriate tool being used. Please refer to the interface manual." )
+            .arg ( APP_NAME ) +
+        TOOLTIP_COM_END_TEXT;
```
Comment (pljones) [inline src/clientsettingsdlg.cpp:229]: Yes indeed, the joy of `make clang_format`.

**Review by ann0see — APPROVED** [review]
Only by quick glance approving it.

**Pull request merged**